src/components/Modal/ModalSocialLogin.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { FcGoogle } from 'react-icons/fc';
+import { RiKakaoTalkFill } from 'react-icons/ri';
+import Button from '../Button/Button';
+import './ModalSocialLogin.scss';
+
+const navigate = url => {
+  window.location.href = url;
+};
+
+const auth = async () => {
+  await fetch('', {
+    method: 'POST',
+  })
+    .then(res => res.json())
+    .then(data => {
+      navigate(data.url);
+    });
+};
```

Comment:
async/await을 좀 더 잘 활용하려면 이런 방식으로 사용하시면 됩니다.
```suggestion
const auth = async () => {
  const response = await fetch('', {
    method: 'POST',
    body: JSON.stringify(...)
  });
  
  const result = await response.json();
  
  if(result.data ...) {
    ...
  }
};
```

Reply:
소셜 로그인은 추가 사항이라 나중에 구현해보겠습니다